### ISSUE TITLE
Markdown support for content editor

### ISSUE THREAD

**Issue by Levitanious** [Levitanious/tempest-cms#3]
There _shall_ be a content editor at some point. Why to reinvent the wheel and make people do weird forum-like half-tags? Markdown is awesome.

**Comment by berserkingyadis** [comment]
agree! im sure there are good libraries for this already. also markdown is awesome

**Comment by Levitanious** [comment]
### You mean...
---
**markdown** `is` _awesome_
---

- [ ] Markdown?
- [x] Awesome.

---

[Awesome](https://github.com/Levitanious/tempest-cms/issues/3).
